src/cmd_all/src/single_node.rs

```diff
@@ -53,6 +56,8 @@ pub static DEFAULT_SINGLE_NODE_STATE_STORE_URL: LazyLock<String> = LazyLock::new
     about = "[default] The Single Node mode. Start all services in one process, with process-level options. This will be executed if no subcommand is specified"
 )]
 /// Here we define our own defaults for the single node mode.
+// TODO(kwannoel): Support user supplied profiles.
+// Perhaps https://docs.rs/clap-serde-derive/0.2.1/clap_serde_derive/?
```

Comment:
I think it's not that necessary. We have lots of configurations in YAML, and they are categorized into storage, compute, server, etc. to make it readable, but we don't want all of them to be settable from CLI.